cli/fathom_web/vectorizer.py

```diff
@@ -278,30 +285,12 @@ def run_in_fathom_fox(*args, desc):
             # than 30s. And once installed once, yarn itself takes only 1.5s to install
             # again. Also, it has security advantages. Though this install itself isn't
             # hashed, so we're just trusting NPM.
-            run_in_fathom_fox('npm', 'install', 'yarn@1.22.4',
+            run_in_fathom_fox(which('npm'), 'install', 'yarn@1.22.4',
```

Comment:
It's funny that this required `which()` but the call to `node` below didn't. Is that on purpose?

Reply:
It is funny. I believe the binaries `npm` and `node` are in the same place, so I would think the `which()` would be required for both or none. I'll check that there isn't anything weird going on with another installation someplace I don't know about.

Reply:
Ah, `node` is an actual executable while `npm` is a `.cmd`.

Reply:
That makes sense, then. Worth a comment so we don't have to wonder about it again.